tests/providerTest.js

```diff
@@ -104,7 +111,9 @@ describe('provider testing', () => {
     defaultOpts.pactBrokerUrl = 'https://jankaritech.pactflow.io'
     defaultOpts.publishVerificationResult = true
     defaultOpts.pactBrokerToken = PACTFLOW_TOKEN
-    defaultOpts.consumerVersionTags = DRONE_SOURCE_BRANCH
+    // when the cron CI build runs, DRONE_SOURCE_BRANCH will empty as it is not a PR/push
+    // so we use DRONE_COMMIT_BRANCH as a fallback
+    defaultOpts.consumerVersionTags = DRONE_SOURCE_BRANCH || DRONE_COMMIT_BRANCH
```

Comment:
Looks like this fix will work. See https://drone.owncloud.com/owncloud/owncloud-sdk/3877/9/11